generate_vcard_qr_code.py

```diff
@@ -30,12 +36,28 @@ def get_vcard_data():
 
     return vcard_data
 
-
-# Generate the vCard data
-data = get_vcard_data()
-
-# Create the QR code
-img = qrcode.make(data)
-img.save("vCardQR.png")
-
-print("vCard QR code generated and saved as 'vCardQR.png'.")
+def generate_qr_code(data):
+    try:
+        # Create the QR code
+        qr = qrcode.QRCode(
+            version=1,
+            error_correction=qrcode.constants.ERROR_CORRECT_L,
+            box_size=10,
+            border=4,
+        )
```

Comment:
If you deviate from the defaults of the library, you shouldn't hard code these in the function call but define these settings somewhere. Would also be nice to have these documented somewhere.

Reply:
Absolutely, it's a good practice to define such settings in a configurable way, especially if they deviate from the library's defaults.